CHANGELOG.md

```diff
@@ -6,13 +6,21 @@ This is a feature release.
 
  * [KIP-848](https://cwiki.apache.org/confluence/display/KAFKA/KIP-848%3A+The+Next+Generation+of+the+Consumer+Rebalance+Protocol): 
    Integration tests running with the new consumer group protocol. The feature is an **Early Access**: not production ready (#1185).
+ * Allow custom http.Client to be passed to SR client (#1099)
```

Comment:
I think we should say "schema registry client" at least for the first time, or maybe just create a different subsection.